index.js

```diff
@@ -0,0 +1,150 @@
+const URL = 'https://ada-backtrek-api.herokuapp.com/trips';
+
+const statusColors = {
+  'success': 'lightgreen',
+  'fail': 'indianred',
+  'loading': 'sandybrown'
+}
+
+const reportStatus = (message, status) => {
+  let color = statusColors[status];
+  return $('#status-message').show(1000).html(message).css('background-color', color);
+};
+
+const clearContent = (idArr) => {
+  idArr.forEach((id) => {
+    $(`#${id}`).empty();
+  });
+};
+
+const reservationForm = ['reservation-name', 'email'];
+const tripForm = ['name', 'continent', 'category', 'weeks', 'cost', 'about'];
+
+const clearForm = (form) => {
+  if (form === 'reservation'){
+    reservationForm.forEach((formField) => {
+      $(`input[name="${formField}"]`).val('');
+    });
+  } else if (form === 'trip') {
+    tripForm.forEach((formField) => {
+      $(`input[name="${formField}"]`).val('');
+    });
+  }
+};
+
+const getFormData = (form) => {
+  let data = {};
+  if (form === 'reservation') {
+    data['name'] = $('input[name="reservation-name"]').val();
+    data['email'] = $('input[name="email"]').val();
+  } else if (form === 'trip'){
+    tripForm.forEach((formField) => {
+      data[formField] = $(`input[name="${formField}"]`).val();
+    });
+  }
+  return data;
+};
+
+const buildTrip = (tripData) => {
+  return (`<div id='${tripData.id}'>
+            <h3>${tripData.name} </h3>
+            <p>Continent: ${tripData.continent} </p>
+            <p>Category: ${tripData.category} </p>
+            <p>Weeks: ${tripData.weeks} </p>
+            <p>Cost: $${tripData.cost} </p>
+            <p>About: ${tripData.about} </p>
+          </div>`);
+};
```

Comment:
Niiiice, I like that this was pulled out into a helper function; even though it's only used once, this part feels appropriately pulled out and separated